cdp/address.py

```diff
@@ -98,6 +103,61 @@ def balances(self):
 
         return BalanceMap.from_models(response.data)
 
+    def historical_balances(self, asset_id) -> Iterator["HistoricalBalance"]:
+        """List historical balances.
+
+        Returns:
+            Iterator[HistoricalBalance]: An iterator of HistoricalBalance objects.
+
+        Raises:
+            Exception: If there's an error listing the historical balances.
+
+        """
+        page = None
+        while True:
+            response: AddressHistoricalBalanceList = Cdp.api_clients.balance_history.list_address_historical_balance(
+                network_id=self.network_id,
+                address_id=self.address_id,
+                asset_id=Asset.primary_denomination(asset_id),
+                limit=100,
+                page=page,
+            )
+
+            for model in response.data:
+                yield HistoricalBalance.from_model(model)
+
+            if not response.has_more:
+                break
+
+            page = response.next_page
+
+    def transactions(self) -> Iterator["Transaction"]:
+        """List transactions of the address.
+
+        Returns:
+            Iterator[Transaction]: An iterator of Transaction objects.
+
+        Raises:
+            Exception: If there's an error listing the transactions.
+
+        """
+        page = None
+        while True:
+            response: AddressTransactionList = Cdp.api_clients.transaction_history.list_address_transactions(
+                network_id=self.network_id,
+                address_id=self.address_id,
+                limit=1,
```

Comment:
```suggestion
                limit=100,
```

Reply:
here intended use 1 as transactions are large, if we get too many together (as I saw before, 5 could be large) it will get error500 with response too large

Reply:
@xinyu-li-cb Makes sense. Thank you for the context!